src/platform/unittest/DataPathTest.cpp

```diff
@@ -238,8 +239,8 @@ struct DataPathTest : public ::testing::TestWithParam<int32_t>
         //
         NextPort = 50000 + (CxPlatCurThreadID() % 10000) + (rand() % 5000);
 
-        LocalIPv4.Resolve(QUIC_ADDRESS_FAMILY_INET, QUIC_TEST_LOOPBACK_FOR_AF(QUIC_ADDRESS_FAMILY_INET));
-        LocalIPv6.Resolve(QUIC_ADDRESS_FAMILY_INET6, QUIC_TEST_LOOPBACK_FOR_AF(QUIC_ADDRESS_FAMILY_INET6));
+        LocalIPv4.Resolve(QUIC_ADDRESS_FAMILY_INET, QUIC_TEST_LOOPBACK_FOR_AF_LOCAL(QUIC_ADDRESS_FAMILY_INET));
+        LocalIPv6.Resolve(QUIC_ADDRESS_FAMILY_INET6, QUIC_TEST_LOOPBACK_FOR_AF_LOCAL(QUIC_ADDRESS_FAMILY_INET6));
```

Comment:
Why make these changes? It looks like you never use the `_REMOTE` version, so I think it'd be best to leave this code (and above) unchanged.

Reply:
An error I saw more than 6 month ago is gone now.